src/plugins/cal-events/cal-events.js

```diff
@@ -4,415 +4,422 @@
-						if ( dayIndex && $days[ dayIndex - 1 ].parentNode.nodeName === "A" ) {
-							$day.wrap( "<a href='javascript:;' class='cal-evt' tabindex='-1'></a>" );
-						} else {
-							$day.wrap( "<a href='javascript:;' class='cal-evt'></a>" );
-						}
-					}
-
-					//Add the event to the list
-					$dayEvents.append( "<li><a tabindex='-1' class='cal-evt-lnk' href='" + event.href + "'>" + event.title + "</a></li>" );
-				}
-			}
-		}
-	},
-
-	filterEvents = function( year, month ) {
-		this.find( "li.cal-disp-onshow" )
-			.addClass( "wb-inv" )
-			.has( ":header[class*=filter-" + year + "-" +
-				wb.string.pad( parseInt( month, 10 ) + 1, 2 ) + "]" )
-			.removeClass( "wb-inv" );
-	},
-
-	showEvents = function() {
-		$( this )
-			.next()
-			.addClass( evDetails );
-	},
-
-	hideEvents = function() {
-		var $link = $( this ),
-			$cell = $link.closest( "td" );
-
-		setTimeout( function() {
-			if ( $cell.find( "a:focus" ).length === 0 ) {
-				$cell.find( "ul" )
-					.removeClass( evDetails )
-					.find( "a" )
-					.attr( "tabindex", "-1" );
-			}
-		}, 5 );
-	};
-
-// Bind the init event of the plugin
-$document.on( "timerpoke.wb " + initEvent, selector, init );
-
-$document.on( "wb-navigate.wb-clndr", ".wb-calevt-cal", function( event ) {
-	var lib = event.target.lib,
-		$calEvent;
-
-	if ( lib ) {
-		$calEvent = lib.$events;
-
-		if ( $calEvent ) {
-			filterEvents.call( $calEvent, event.year, event.month );
-
-			//TODO: Added for backwards compatibility. Remove in previous versions
-			$calEvent.trigger( "wb-updated" + selector );
-		}
-	}
-} );
-
-$document.on( "focusin focusout keydown", ".wb-calevt-cal .cal-days td > a", function( event ) {
-	var eventType = event.type,
-		$link;
-
-	switch ( eventType ) {
-		case "focusin":
-			showEvents.call( event.target );
-			break;
-		case "focusout":
-			hideEvents.call( event.target );
-			break;
-		case "keydown":
-			$link = $( event.target );
-			if ( ( event.which === 13 || event.which === 32 ) && $link.hasClass( "cal-evt" ) ) {
-				$( event.target ).next().find( "a:first" ).trigger( setFocusEvent );
-			}
-			break;
-	}
-} );
-
-$document.on( "keydown", ".wb-calevt-cal .cal-days td > ul li", function( event ) {
-	var $item = $( event.currentTarget ),
-		$toFocus, $itemParent;
-
-	switch ( event.which ) {
-		case 38:
-			$toFocus = $item.prev().find( "a" );
-			if ( $toFocus.length === 0 ) {
-				$toFocus = $item.siblings( ":last" ).find( "a" );
-			}
-			$toFocus.trigger( setFocusEvent );
-			break;
-		case 40:
-			$toFocus = $item.next().find( "a" );
-			if ( $toFocus.length === 0 ) {
-				$toFocus = $item.siblings( ":first" ).find( "a" );
-			}
-			$toFocus.trigger( setFocusEvent );
-			break;
-		case 27:
-			$itemParent = $item.closest( "td" ).children( "a" );
-			$itemParent.trigger( setFocusEvent );
-			break;
-	}
-} );
-
-// Add the timer poke to initialize the plugin
-wb.add( selector );
-
-} )( jQuery, window, wb );
+(function($, window, wb) {
+    "use strict";
```

Comment:
@nschonni @LaurentGoderre 
Not sure what I need to do...
I did a HTML prettify for cal-events.js because I had this error when grunt dist:
"error Mixed spaces and tabs no-mixed-spaces-and-tabs".
Do I need to get the previous format?
Thanks.